src/main/java/ecs/CharacterController.java

```diff
@@ -2,107 +2,98 @@
 
 import input.Keyboard;
 import org.joml.Vector2f;
+import physics.force.Force;
 
-import java.util.function.Consumer;
-
-//TODO this class is just made to support topDown and platformer demo scenes - its not good at all
+//this class is just made to support topDown and platformer demo scenes - its not good at all!!!!
```

Comment:
This is true, it will be removed before the 1.0 release.

Reply:
ye